src/openkit-shared/API/LogLevel.cs

```diff
@@ -0,0 +1,69 @@
+﻿//
+// Copyright 2018-2019 Dynatrace LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+using System;
+using System.Linq;
+
+namespace Dynatrace.OpenKit.API
+{
+    /// <summary>
+    /// Log level enumeration.
+    /// </summary>
+    /// <remarks>
+    /// This level is used to specify the priority of a log event.

```

Comment:
severity instead of priority

Reply:
In logging frameworks, you typically use the term "priority".
"severe" might be found in some frameworks to log a severe message.